tools/android_custom_build/readme.txt

```diff
@@ -0,0 +1,10 @@
+This directory contains helper files for building a custom ONNX Runtime
+Android package. It can be copied outside of the ONNX Runtime repo and used
+independently.
+
+Run the ./build_custom_android_package.py script. Use the --help option for
+more information.
+
```

Comment:
Could we make this an md file?

Should we also include a step-by-step here or will that be added in the docs and we could link to it instead?

e.g. 
  - example of running model conversion script to get config to use
  - update build settings to enable type reduction/NNAPI if applicable
  - example command line to run with docker and copy out the aar
  - info or link to doco on how to use that in an android app

Reply:
added a link - there's a section there that we can add this information to:
https://onnxruntime.ai/docs/build/custom.html#android